grakn-graql/src/main/java/ai/grakn/graql/internal/query/ConceptBuilder.java

```diff
@@ -90,14 +90,38 @@
     private final Map<BuilderParam<?>, Object> params = new HashMap<>();
```

Comment:
I think I realised why this change it confusing to follow. Maybe this variable can be renamed to `expectedParams`. My justification for this is the following:

```
params.forEach((param, value) -> {
    if (!usedParams.contains(param)) {
        throw GraqlQueryException.insertUnexpectedProperty(param.name(), value, concept);
    }
});
```

This will make more sense as:

```
expectedParams.forEach((expectedParam, value) -> {
    if (!usedParams.contains(expectedParam)) {
        throw GraqlQueryException.insertUnexpectedProperty(param.name(), value, concept);
    }
});
```

Then it's clear that you used something that was not expected so BOOM. What do you think?